clusterloader2/testing/watch-list/config.yaml

```diff
@@ -1,3 +1,4 @@
+{{$enableWatchListFeature := DefaultParam .ENABLE_WATCH_LIST_FEATURE false}}
```

Comment:
you could change name to something with `CL2_` prefix, like this: `CL2_ENABLE_WATCH_LIST_FEATURE`.
Then you don't need `overrides/watch_list.yaml` file and can just specify `- --env=CL2_ENABLE_WATCH_LIST_FEATURE=true` in job spec :)

Reply:
+1

Reply:
pushed, PTAL.